app/views/user_mailer/reset_password_instructions.html.erb

```diff
@@ -3,17 +3,26 @@
   <table>
     <tbody>
     <tr>
-      <td width="200">
-        <%= image_tag('letter-warning.svg', width: 200, height: 200, alt: '') %>
+      <td width="140">
+        <%= image_tag('letter-warning.svg', width: 140, height: 140, alt: '') %>
       </td>
-      <td>
-        <h3><%= t('user_mailer.reset_password_instructions.gpo_letter_header') %></h3>
+      <td style="padding-left: 14px;">
```

Comment:
Is there a reason to hardcode styling here? Maybe we should create a class in css.

Reply:
From the Figma design, it looks like this should be 24px, which would neatly map to 3 units in the design system.

```suggestion
      <td class="padding-left-3">
```

Ref: https://designsystem.digital.gov/design-tokens/spacing-units/

Reply:
Ah yes, good catch. I will need to double check that this works -- there is a gem that is "magically" inserting inline styles to various elements in these mailers, which theoretically could override any classes